examples/multi-backend-multi-mig-bucket-https-lb/main.tf

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "group1_region" {
```

Comment:
Please move variables and outputs to appropriate files

Reply:
@ingwarr, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-lb-http/compare/3863b4b4031bee488cf54f700867f67fea71b878..faaadf20455456bce10b8910bd867134b3a70dbe)